### PR TITLE
fix(locales) Fixing mainly french translations

### PR DESCRIPTION
**Pull request by Mathu-lmn**
Fixing main french translations issue ;)

### CODE REVIEW

**Comment by Mathu-lmn** [comment]
@Mycroft-Studios Hey, do you have time to review this PR ? I think it's quite interesting ;)

**Comment by Benzo00** [comment]
take a sec and read this https://github.com/esx-framework/esx-legacy/issues/343 and fix the title :)

File: [esx_addons]/esx_property/locales/fr.lua [hunk 1]
```diff
@@ -30,8 +30,8 @@ Locales['fr'] = {
   ['press_to_exit'] = 'appuyez sur [E] pour sortir de la propriété',
   ['not_enough'] = 'vous n\'avez pas assez d\'argent',
   ['invalid_quantity'] = 'quantité invalide',
-  ['paid_rent'] = 'you paid your rent at $%s for %s',
-  ['paid_rent_evicted'] = 'you have been ~o~evicted from %s for not affording rent at $%s',
+  ['paid_rent'] = 'vous avez payé votre loyer à $%s pour %s',
```
Comment (jbdevic) [inline [esx_addons]/esx_property/locales/fr.lua:33]: "Vous avez payé votre loyer **de** XX$ pour XXX"
Determinant "De" seems more appropriate in this situation

File: [esx_addons]/esx_property/locales/fr.lua [hunk 2]
```diff
@@ -30,8 +30,8 @@ Locales['fr'] = {
...
+  ['paid_rent_evicted'] = 'vous avez été ~o~expulsé de %s pour ne pas avoir payé un loyer à $%s',
```
Comment (jbdevic) [inline [esx_addons]/esx_property/locales/fr.lua:34]: Same here

**Review by jbdevic — CHANGES_REQUESTED** [review]
PR is good.

Looking at the files I saw other errors so it would be necessary to make new changes.
Also, shouldn't we change the currency to €?

**Pull request merged**